corehq/apps/app_manager/models.py

```diff
@@ -329,12 +334,17 @@ def all_property_names(self):
         return names
 
 
+class ParentIndex(DocumentSchema):
```

Comment:
I see it does tie in with `action.parents` but just wondering if its worth moving away from 'parent' metaphor in the models and sticking closer to language of the XML spec i.e. index / indices